code/modules/projectiles/guns/specialist.dm

```diff
@@ -1136,3 +1136,35 @@ obj/item/weapon/gun/minigun/Fire(atom/target, mob/living/user, params, reflex =
 		return in_chamber ? 1 : 0
 	else
 		return in_chamber ? (current_mag.current_rounds + 1) : current_mag.current_rounds
+
+//railgun
+
+//-------------------------------------------------------
+//T-81 Auto-Sniper
+
+/obj/item/weapon/gun/rifle/railgun
+	name = "\improper TX-220 railgun"
+	desc = "The TX-220 is a specialized heavy duty railgun made to shred through hard armor to allow for follow up attacks. Uses specialized canisters to reload."
+	icon = 'icons/Marine/gun64.dmi'
+	icon_state = "railgun"
+	item_state = "railgun"
+	max_shells = 1 //codex
+	caliber = "rail projectile"
+	fire_sound = 'sound/weapons/guns/fire/sniper.ogg'
```

Comment:
ew

Reply:
i might make unique sounds in a separate PR.

Reply:
Actually, why not I just add it here. License is on CC-BY-NC-SA 3.0, same as every other PR that I have with remixed sounds from other sources.